yarn-project/telemetry-client/src/otel_resource.ts

```diff
@@ -11,7 +12,14 @@ import { aztecDetector } from './aztec_resource_detector.js';
 
 export async function getOtelResource(): Promise<IResource> {
   const resource = detectResourcesSync({
-    detectors: [osDetectorSync, envDetectorSync, processDetectorSync, serviceInstanceIdDetectorSync, aztecDetector],
+    detectors: [
+      osDetectorSync,
+      envDetectorSync,
+      processDetectorSync,
+      serviceInstanceIdDetectorSync,
+      gcpDetector,
```

Comment:
Should we gate this gcp detector behind Mitch's GCP observability flag, once that's merged?

Reply:
I can comment it out for now. cc @just-mitch